dictionaries/en_shared/src/shared-additional-words.txt

```diff
@@ -23,7 +23,7 @@ Christoph
 cohabitate
 cohabitated
 cohabitates
-cohabitating
```

Comment:
I think this link was used as a reference: https://en.wiktionary.org/wiki/cohabitate

But, sometimes Wiktionary feels a bit auto-generated.

Reply:
See also: https://books.google.com/ngrams/graph?content=cohabitating%2Ccohabiting%2Ccohabitate%2Ccohabit&year_start=1980&year_end=2022&corpus=en&smoothing=3

It use has been going up. I couldn't find the PR that introduce this change.

Reply:
> But, sometimes Wiktionary feels a bit auto-generated.

I think so too.

Do you think `cohabitating` should be kept then ?

I'm unsure about your opinion here.

Reply:
cohabitate and its variations comes from this PR

- https://github.com/streetsidesoftware/cspell-dicts/pull/2655

A list of words extracted from Kubernetes code. The words exist in Wiktionary, but as you said. They sound a bit auto generated.

cohabitate and its variations exist, but I agree we should suggest replacing them by cohabit ones.

Reply:
> cohabitate and its variations exist, but I agree we should suggest replacing them by cohabit ones.

I changed my mind and feel like `cohabit` and `cohabitate` can well ... `cohabit`

I feel its usage is more about people sharing rent for the same place, either if they are in couple or not.

So I don't think we should suggest replacing cohabitate variations to cohabit ones.

So I'm closing this thread